### ISSUE TITLE
Uniform access to data via download button

### ISSUE THREAD

**Issue by agricolamz** [clld/clics#14]
Hi, 

I tried to download data from concept pages (e. g. [DUST](https://clics.clld.org/parameters/2)), but its .csv version lacks the information about languages, and the .xls version lacks the `source_form` column. There is also lack of language information in the table on the site (but this probably should be a different issue).

**Comment by xrotwang** [comment]
Yes, I agree that at present the download options offered by the CLICS clld app are lacking. The main reason for this is that I feel with CLDF, the functionality of clld as providing uniform access to the raw data has been largely superseded. So I guess any solution of this issue will rather go towards doing away with the download options in clld and emphasizing where the CLDF data can be accessed, rather than fleshing out the clld download functionality.

**Comment by chrzyki** [comment]
Would this be a direct link to the corresponding repository on GitHub, then?

**Comment by xrotwang** [comment]
For the clics app, it would rather be a link to https://github.com/clics/clics2/blob/master/README.md
